pywr/solvers/cython_glpk_edge.pyx

```diff
@@ -6,24 +6,21 @@ cimport numpy as np
 
 cimport cython
 
+from .glpk cimport GLPKSolver, BasisManager, simplex, set_obj_coef, set_row_bnds, set_col_bnds, set_mat_row
+
 from pywr._core import BaseInput, BaseOutput, BaseLink
 from pywr._core cimport *
 from pywr.core import ModelStructureError
 import time
 
 include "glpk.pxi"
```

Comment:
Is there a reason to keep `glpk.pxi` at all? Why can't all the function definitions be in the `glpk.pxd`?